src/app/zap-templates/zcl/data-model/chip/camera-av-stream-management-cluster.xml

```diff
@@ -0,0 +1,578 @@
+      <arg id="0" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+      <arg id="1" name="AudioCodec" type="AudioCodecEnum" min="0x00" max="0x01"/>
+      <arg id="2" name="ChannelCount" type="int8u" default="1"/>
+      <arg id="3" name="SampleRate" type="int32u"/>
+      <arg id="4" name="BitRate" type="int32u"/>
+      <arg id="5" name="BitDepth" type="int8u"/>
+    </command>
+
+    <command code="0x01" source="server" name="AudioStreamAllocateResponse" optional="true" disableDefaultResponse="true">
+      <description>This command SHALL be sent by the camera in response to the AudioStreamAllocate command, carrying the newly allocated audio stream identifier.</description>
+      <arg id="0" name="AudioStreamID" type="int16u"/>
+    </command>
+
+    <command code="0x02" source="client" name="AudioStreamDeallocate" optional="true">
+      <description>This command SHALL deallocate an audio stream on the camera, corresponding to the given audio stream identifier.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="0" name="AudioStreamID" type="int16u"/>
+    </command>
+
+    <command code="0x03" source="client" name="VideoStreamAllocate" optional="true" response="VideoStreamAllocateResponse">
+      <description>This command SHALL allocate a video stream on the camera and return an allocated video stream identifier.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="1" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+      <arg id="2" name="VideoCodec" type="VideoCodecEnum" min="0x00" max="0x03"/>
+      <arg id="3" name="MinFrameRate" type="int16u"/>
+      <arg id="4" name="MaxFrameRate" type="int16u"/>
+      <arg id="5" name="MinResolution" type="VideoResolutionStruct"/>
+      <arg id="6" name="MaxResolution" type="VideoResolutionStruct"/>
+      <arg id="7" name="MinBitRate" type="int32u"/>
+      <arg id="8" name="MaxBitRate" type="int32u"/>
+      <arg id="9" name="MinFragmentLen" type="int16u"/>
+      <arg id="10" name="MaxFragmentLen" type="int16u"/>
+    </command>
+
+    <command code="0x04" source="server" name="VideoStreamAllocateResponse" optional="true" disableDefaultResponse="true">
+      <description>This command SHALL be sent by the camera in response to the VideoStreamAllocate command, carrying the newly allocated video stream identifier.</description>
+      <arg id="0" name="VideoStreamID" type="int16u"/>
+    </command>
+
+    <command code="0x05" source="client" name="VideoStreamModify" optional="true">
+      <description>This command SHALL be used to modify the resolution of a stream specified by the VideoStreamID.</description>
+      <access op="invoke" privilege="administer"/>
+      <arg id="0" name="VideoStreamID" type="int16u"/>
+      <arg id="1" name="Resolution" type="VideoResolutionStruct"/>
+    </command>
+
+    <command code="0x06" source="client" name="VideoStreamDeallocate" optional="true">
```

Comment:
This command allows one client to deallocate a stream a different client, possibly on a different fabric, allocated, right?  Is this intentional?

Reply:
Yes, this cluster is a global cluster across fabrics. And the streams are shared/reused. Admins are allowed to de-allocate provided the ref-count on the stream is 0.